### ISSUE TITLE
[Bug]  The formula of MSLL, one of the evaluation indicators of GP, is implemented incorrectly.

### ISSUE THREAD

**Issue by jongwonKim-1997** [cornellius-gp/gpytorch#2174]
# 🐛 Bug

The Gaussian process for machine learning cited in the GPytorch documentation expresses MSLL as follows;

![image](https://user-images.githubusercontent.com/77820101/197520662-86d084e5-a179-403f-8bc5-8465f404c7cc.png)

The current implementation code in GPytorch is below and I think 0.5 should be multiplied in parentheses.

```python
def mean_standardized_log_loss(
    pred_dist: MultivariateNormal,
    test_y: torch.Tensor,
):
    """
    Mean Standardized Log Loss.
    Reference: Page No. 23,
    Gaussian Processes for Machine Learning,
    Carl Edward Rasmussen and Christopher K. I. Williams,
    The MIT Press, 2006. ISBN 0-262-18253-X
    """
    combine_dim = -2 if isinstance(pred_dist, MultitaskMultivariateNormal) else -1
    f_mean = pred_dist.mean
    f_var = pred_dist.variance
    return 0.5 * (torch.log(2 * pi * f_var) + torch.square(test_y - f_mean) / (2 * f_var)).mean(dim=combine_dim)

```

I don't know where to report this little error, so I'm posting it here.

**Comment by gpleiss** [comment]
You're right - the 0.5 should be moved inside the parenthesis. Can you open a PR to fix this?

**Comment by jongwonKim-1997** [comment]
As you said, I opened PR.

Thank you for making such an excellent GPytorch framework. Thank you for always helping my research and motivating me academically.

**Comment by gpleiss** [comment]
Closed by #2177. Thanks @jongwonKim-1997 !
